tests/test_ops_regrid.py

```diff
@@ -21,8 +21,11 @@ def _check_output_nc(result, fname="output_001.nc"):
     assert fname in [os.path.basename(_) for _ in result]
 
 
+@pytest.mark.xfail(
+    reason="Needs xarray fix https://github.com/pydata/xarray/issues/7794"
+)
 @pytest.mark.skipif(xe is None, reason=XESMF_IMPORT_MSG)
-def test_regrid_basic(tmpdir, tmp_path, mini_esgf_data, xfail_if_xarray_incompatible):
```

Comment:
Is the `xfail_if_xarray_incompatible` fixture obsolete now? If so, we should remove it from `tests/conf.py`

Reply:
done 👍